org_status/status_providers/travis.py

```diff
@@ -6,13 +7,19 @@
 class TravisBuildStatus(StatusProvider):
     BadgeTemplate = ('https://api.travis-ci.org/{group}'
                      '/{repo}.svg?branch={branch}')
+    TRAVIS_STATUS_URL = ('https://pnpcptp8xh9k.statuspage.io/'
+                         'api/v2/status.json')
 
     def get_status(self, repo, host, branch='master'):
         badge_result = requests.get(self.get_badge_url(repo,
                                                        host,
                                                        branch=branch))
 
+        response = requests.get(cls.TRAVIS_STATUS_URL)
+        status = json.loads(response.text)
+
         if badge_result.status_code != 200:
             return Status.UNDETERMINED
 
-        return self.status_from_badge_svg(badge_result.text)
+        return (self.status_from_badge_svg(badge_result.text), status['status']['description'] == 'All Systems Operational')
```

Comment:
You shouldn't be doing this here. Ideally we should have check this when aggregating the results and it should be done only once.